.github/workflows/package-build-ofed.yml

```diff
@@ -183,23 +189,31 @@ jobs:
         run: |
           source venvs/kayobe/bin/activate &&
           source src/kayobe-config/kayobe-env --environment ci-builder &&
-          kayobe playbook run src/kayobe-config/etc/kayobe/ansible/growroot.yml
+          kayobe playbook run src/kayobe-config/etc/kayobe/ansible/growroot.yml \
+          -e seed_bootstrap_user="cloud-user" \
+          -e controller_bootstrap_user="cloud-user" \
         env:
           KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
 
       - name: Configure the seed host (Builder VM)
         run: |
           source venvs/kayobe/bin/activate &&
           source src/kayobe-config/kayobe-env --environment ci-builder &&
-          kayobe seed host configure --skip-tags network,docker
+          kayobe seed host configure \
+          --skip-tags network,docker,docker-registry \
+          -e seed_bootstrap_user="cloud-user" \
+          -e controller_bootstrap_user="cloud-user" \
+          -e dnf_install_doca=true \
+          -e dnf_enable_doca_modules=false \
+          -e stackhpc_repos_enabled=true
```

Comment:
It would be nice to get these into the kayobe config rather than relying on extra vars.
If we can't add them to the builder env, what are your thoughts on adding a new doca-builder env which is dependent on the builder env? (see https://docs.openstack.org/kayobe/latest/multiple-environments.html#environment-dependencies) for more details on nested environments